modules/core/src/lib/layer.js

```diff
@@ -159,6 +160,11 @@ export default class Layer extends Component {
     return this.internalState && this.internalState.layer;
   }
 
+  // Returns the default parse options for async props
+  getLoadOptions() {
+    return this.props.loadOptions || {};
```

Comment:
Nit: To make it harder to write bad redefinitions of this method, I would apply the default empty object when we call this function, rather than inside the function (if it is needed at all).